chain/cosmos/chain_node.go

```diff
@@ -515,13 +515,19 @@ func (tn *ChainNode) TxCommand(keyName string, command ...string) []string {
 		if command[i] == "--fees" {
 			feesFound = true
 		}
+		if command[i] == "--gas" {
+			gasFound = true
+		}
 	}
 	if !gasPriceFound && !feesFound {
 		command = append(command, "--gas-prices", tn.Chain.Config().GasPrices)
 	}
 	if !gasAdjustmentFound {
 		command = append(command, "--gas-adjustment", strconv.FormatFloat(tn.Chain.Config().GasAdjustment, 'f', -1, 64))
 	}
+	if !gasFound && !feesFound && tn.Chain.Config().Gas != "" {
+		command = append(command, "--gas", tn.Chain.Config().Gas)
+	}
```

Comment:
How does this affect tx commands which are not actual txs? (i.e. appd tx decode <amino_encoded>

May want to add a test for this in [chain_miscellaneous_test.go](https://github.com/strangelove-ventures/interchaintest/blob/9b9c2db326d6ec628db7b035be23eb1e7edfc441/examples/cosmos/chain_miscellaneous_test.go#L27)

Reply:
done!